tests/layer1/test_layer1_speed_Novus_100G.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+from abstract_open_traffic_generator.port import *
+from abstract_open_traffic_generator.config import *
+from abstract_open_traffic_generator.layer1 import *
+from abstract_open_traffic_generator.control import *
+
+
+@pytest.mark.e2e
+def test_layer1(api, options, utils):
+    """Test that layer1 configuration settings are being applied correctly
+    A user should be able to configure ports with/without locations.
+    The expectation should be if a location is configured the user wants to
+    connect but debug should allow for config creation without location.
+    Ports with no location should not generate an error message.
+    Ports with location should generate an error message if unable to connect.
+
+    Validation: Validate the layer1 properties applied using Restpy
+    """
+    speed_type = {'speed_10_gbps': '10000',
+                  'speed_25_gbps': '25000',
+                  'speed_40_gbps': '40000',
+                  'speed_100_gbps': '100000'}
+    port1 = Port(name='port1', location=utils.settings.ports[0])
+    port2 = Port(name='port2', location=utils.settings.ports[1])
+    speed = utils.settings.speed
+    auto_negotiate = True
+    ieee_media_defaults = False
```

Comment:
there should be at least one use case where ieee_media_defaults should be set to True. In such cases, one need not pass following:
- ieee_media_defaults
- auto_negotiate
- AutoNegotiation

Reply:
`ieee_media_defaults` is True as per the model. I felt this is covered in all other tests, we can discuss tomorrow and conclude